sharepoint/sharepoint-server-ps/sharepoint-server/Set-SPWebApplication.md

```diff
@@ -222,14 +277,31 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -Confirm
-Prompts you for confirmation before executing the command.
-For more information, type the following command: `get-help about_commonparameters`
+### -Identity
+Specifies the name or URL of the Web application.
+
+The type must be a valid name, in the form WebApplication-1212, or URL, in the form http://server_name.
```

Comment:
Rather than using http://server_name, could you use https://example.contoso.com? In my personal opinion, this encourages SSL + FQDN usage, guiding administrators towards best practice.